pkg/kubelet/certificate/certificate_store_test.go

```diff
@@ -278,7 +280,9 @@ func TestLoadFile(t *testing.T) {
 	}()
 
 	pairFile := filepath.Join(dir, "kubelet-pair.pem")
-	if err := ioutil.WriteFile(pairFile, []byte(certificateData+"\n"+privateKeyData), 0600); err != nil {
+	data := append(testCertificates[0].certificatePEM, []byte("\n")...)
+	data = append(data, testCertificates[0].keyPEM...)
```

Comment:
Oh, sorry, should be `byte('\n')` (single quotes). I forgot that `...` parameters can't be mixed with other params though, so I guess you still need the 2 lines (or nested append)